app-emacs/dape/dape-0.17.0.ebuild

```
@@ -0,0 +1,66 @@
+# Copyright 2023-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+NEED_EMACS=29.1
+
+inherit elisp
+
+DESCRIPTION="Debug Adapter Protocol for Emacs"
+HOMEPAGE="https://github.com/svaante/dape"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/svaante/${PN}.git"
+else
+	SRC_URI="https://github.com/svaante/${PN}/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="GPL-3+"
+SLOT="0"
+
+RDEPEND="
+	|| (
+		app-emacs/jsonrpc
+		>=app-editors/emacs-30:*
+	)
+"
+DEPEND="
+	${RDEPEND}
+"
+
+# Requires debugpy, but we do not package debugpy right now, as well as
+# js-debug.  Most tests use only the former.
+RESTRICT="test"
+
+DOCS=( README.org CHANGELOG.org LICENSE )
+SITEFILE="50${PN}-gentoo.el"
+
+# elisp-enable-tests ert . -l dape-tests.el
+
+pkg_setup() {
```

You should still call `elisp_pkg_setup` which checks that the Emacs executable is sane (e.g. not XEmacs or some Microemacs).

```suggestion
pkg_setup() {
	elisp_pkg_setup
```

---

ah, yeah, good point, will add and push